.github/contributing.md

```diff
@@ -97,6 +97,10 @@
   draft PR to a regular PR, you can revert it to a draft PR by clicking
   "Convert to draft" under the list of reviewers in the right sidebar of the
   "Conversation" view.
+- Use `auto-retest` label if you want CI to rerun E2E test on failures.
```

Comment:
```suggestion
- Use the `auto-retest` label if you want CI to re-run E2E test on failures.
```

Reply:
Maybe mention the max number of retries?

Reply:
Moreover, it took me some time to notice that writing `/retest-times 20 sth` on one day and then `/retest-times 20 sth` few days later will not give me 40 reruns, because all comments in the PR are counted and we do not look how many `/retest` comments were posted before and how many after a `/retest-times` comment. It would be cool to write about that too ;)

Reply:
How about giving a link to the source? I'm afraid that doc could get stale and nobody will remember to update it

Reply:
+1 to adding a link. Maybe just warn that some caveats might apply w.r.t. how things are calculated.